data/conf/epggrab/opentv/dict/skynz

```diff
@@ -645,187 +645,187 @@
   },
   {
     "code": "101010111000110000110010111",
-    "data": "�"
+    "data": "Ў"
```

Comment:
This entire file looks broken, at least when viewing it on GitHub

Reply:
Because is on local encoding not UTF-8.

Hebrew is total unreadable on any other local encoding even on github without conversion.

![image](https://github.com/tvheadend/tvheadend/assets/16975883/55536021-bb0a-4429-839a-99fc051e2389)

Reply:
Well what you are showing in the screenshot is still wrong, this is supposed to be the upside down exclamation mark...

Have you tested your PR in practice? Or just converted the encoding without testing what effect it has?

Reply:
In screenshot you see pr where on left is old local encoding on right utf-8


other source of same dictionary
https://raw.githubusercontent.com/jahutchi/openTVtoXML/standalone/providers/skynz_160.0.dict

Reply:
And on the right side it is still wrong.

Unless you come up with a good reason why we should switch the encoding I'm going to close this as it's clearly breaking things and apparently you haven't tested your changes either.